src/apps/dashboard/routes/logs/index.tsx

```diff
@@ -0,0 +1,132 @@
+            setLoading(false);
+        }
+    }, [isConfigurationPending, defaultConfiguration]);
+
+    const setLogWarningMessage = useCallback((_: ChangeEvent<HTMLInputElement>, checked: boolean) => {
+        setConfiguration({
+            ...configuration,
+            EnableSlowResponseWarning: checked
+        });
+    }, [configuration]);
+
+    const onResponseTimeChange = useCallback((event: ChangeEvent<HTMLTextAreaElement>) => {
+        setConfiguration({
+            ...configuration,
+            SlowResponseThresholdMs: parseInt(event.target.value, 10)
+        });
+    }, [configuration]);
+
+    const onSubmit = useCallback(() => {
+        setIsSubmitting(true);
+    }, []);
+
+    if (isLogEntriesPending || isConfigurationPending || loading || !logs) {
+        return <Loading />;
+    }
+
+    return (
+        <Page
+            id='logPage'
+            title={globalize.translate('TabLogs')}
+            className='mainAnimatedPage type-interior'
+        >
+            <Box className='content-primary'>
+                <Form method='POST' onSubmit={onSubmit}>
+                    <Stack spacing={3}>
+                        <Typography variant='h1'>
+                            {globalize.translate('TabLogs')}
+                        </Typography>
+
+                        {isSubmitting && actionData?.isSaved && (
+                            <Alert severity='success'>
+                                {globalize.translate('SettingsSaved')}
+                            </Alert>
+                        )}
+
+                        <FormControlLabel
+                            control={
+                                <Switch
+                                    checked={configuration?.EnableSlowResponseWarning}
+                                    onChange={setLogWarningMessage}
+                                    name={'EnableWarningMessage'}
+                                />
+                            }
+                            label={globalize.translate('LabelSlowResponseEnabled')}
+                        />
+
+                        <TextField
+                            fullWidth
+                            type='number'
+                            name={'SlowResponseTime'}
+                            label={globalize.translate('LabelSlowResponseTime')}
+                            value={configuration?.SlowResponseThresholdMs}
+                            disabled={!configuration?.EnableSlowResponseWarning}
+                            onChange={onResponseTimeChange}
+                        />
+
+                        <Button
+                            type='submit'
+                            size='large'
+                        >
+                            {globalize.translate('Save')}
+                        </Button>
+                    </Stack>
+                </Form>
+                <Box className='serverLogs readOnlyContent' sx={{ mt: 1 }}>
```

Comment:
```suggestion
                <Box className='serverLogs readOnlyContent' sx={{ mt: 3 }}>
```

We use 3 for the form spacing, so it probably makes sense to use the same here.